app/src/main/java/com/nwa/smartgym/activities/SportSchedule.java

```diff
@@ -0,0 +1,104 @@
+package com.nwa.smartgym.activities;
+
+import android.app.ProgressDialog;
+import android.content.Intent;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.Toast;
+
+import com.nwa.smartgym.R;
+import com.nwa.smartgym.api.ServiceGenerator;
+import com.nwa.smartgym.api.SportScheduleAPI;
+import com.nwa.smartgym.lib.SportScheduleAdapter;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.UUID;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class SportSchedule extends AppCompatActivity {
+
+    UUID userId = UUID.fromString("6159c216-a92f-4fdb-b043-baefa82009f1");
```

Comment:
Weet hij altijd in de api de request.user.id dan? Als je bent ingelogd dan natuurlijk?